addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5856,6 +5856,17 @@ const ImageHandlerOption = SnippetOptionWidget.extend({
             // above the original format one of the same size.
             widths[maxWidth - 0.1] = [sprintf(_t("%spx"), maxWidth), 'image/webp'];
         }
+        const currentMimetype = this._getImageMimetype(img);
```

Comment:
If not already listed, the format "img.naturalWidth - currentMimetype " has to be added as this is the one that will be selected by `selectFormat`